dcbr-api/api/views.py

```diff
@@ -3,25 +3,36 @@
 from rest_framework.views import APIView
 from rest_framework import viewsets
 from api.serializers import (
+    Registration_Number_Serializer,
     Operator_Serializer,
     Address_Serializer,
-    Risk_Factor_Animal_Serializer,
-    Risk_Factor_Operation_Serializer,
+    Animal_Risk_Factor_Serializer,
+    Operation_Risk_Factor_Serializer,
     Inspection_Serializer,
     Association_Membership_Serializer,
 )
 
 # from api.models import Category, Entry, Operator, Address
 from api.models import (
+    Registration_Number,
     Operator,
     Address,
     Inspection,
-    Risk_Factor_Animal,
-    Risk_Factor_Operation,
+    Animal_Risk_Factor,
+    Operation_Risk_Factor,
     Association_Membership,
 )
 
 
+class Registration_Number_ViewSet(viewsets.ModelViewSet):
```

Comment:
I am a bit confused by the naming here: was this meant to be something like `Operator_Registration` rather than `Registration_Number`? Based on how it is called from the VUE code it looks to me like we could name this better.

Reply:
The naming is to allow the admin interface to make sense to the admin users, since this is their only way to manually entering paper registrations